iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -211,6 +211,15 @@ <h2 data-i18n="inspector.validationResults">Validation results</h2>
           </div>
       </div>
     </div>
+    <div class="dialog text-block-viewer">
+      <div class="close"></div>
+      <div class="contents">
+        <h2 class="title"></h2>
+        <iframe></iframe>
```

Comment:
You might give this an id. When working on the plugin example there wasn't a good way to get the iframe without a jquery wrapper. It'd be cool to be able to select by id instead

Reply:
Done in 5d5f454